views/profile/_blocks.tt

```diff
@@ -109,9 +109,7 @@
   [% dw.img( 'id_openid', '' ); "[$s] "; %]
     [%- site_users = [];
         i = 0;
-        i_max = openid_info.sites.$s.size;
-        WHILE i < i_max;
-          user = openid_info.sites.$s.$i;
+        FOREACH user IN openid_info.sites.$s;
           text = openid_info.shortnames.$s.$i;
```

Comment:
Hmm, and we're sure the $i will always line up with the index of the user we're tracking? (It looks probably fine, I just have an innate heeby-jeeby reaction to tracking two arrays in parallel like that. I bet that's also just logic translated verbatim from the BML conversion.)

Reply:
Yeah, okay, I read the parse_openids function, it should be fine. 👍🏼

Reply:
Yup, that's the one thing I don't like about this change - it makes the parallelism of the two arrays less obvious to the casual observer. But it doesn't seem worth reconstructing the entire underlying data structure to make it clearer.